server/src-test/Hasura/Server/MigrateSpec.hs

```diff
@@ -62,20 +63,31 @@ spec
 spec pgConnInfo = do
   let dropAndInit time = CacheRefT $ flip modifyMVar \_ ->
         dropCatalog *> (swap <$> migrateCatalog time)
+      upgradeToLatest time = CacheRefT $ flip modifyMVar \_ ->
+        swap <$> migrateCatalog time
+      downgradeTo v = downgradeCatalog DowngradeOptions{ dgoDryRun = False, dgoTargetVersion = v }
```

Comment:
Is there any reason not to keep them with the other similar actions like this? Others were only used once and are now used multiple times, and that could easily happen with these too. It's not like it's not easily discoverable.

Reply:
I just tend to think that it’s helpful to keep the scope of a binding as local as possible, especially in the context of test cases. But this isn’t a terribly important point; if you feel otherwise, feel free to leave it as it is.